diff --git a/node_modules/commit-and-tag-version/index.js b/node_modules/commit-and-tag-version/index.js
index ecb1934..53070cf 100755
--- a/node_modules/commit-and-tag-version/index.js
+++ b/node_modules/commit-and-tag-version/index.js
@@ -84,9 +84,10 @@ module.exports = async function standardVersion(argv) {
     }
 
     const newVersion = await bump(args, version);
-    await changelog(args, newVersion);
+    const changelogOutput = await changelog(args, newVersion);
     await commit(args, newVersion);
     await tag(newVersion, pkg ? pkg.private : false, args);
+    return { newVersion, changelogOutput };
   } catch (err) {
     printError(args, err.message);
     throw err;
diff --git a/node_modules/commit-and-tag-version/lib/lifecycles/changelog.js b/node_modules/commit-and-tag-version/lib/lifecycles/changelog.js
index fcdd13b..fd2a780 100644
--- a/node_modules/commit-and-tag-version/lib/lifecycles/changelog.js
+++ b/node_modules/commit-and-tag-version/lib/lifecycles/changelog.js
@@ -11,8 +11,9 @@ const START_OF_LAST_RELEASE_PATTERN =
 async function Changelog(args, newVersion) {
   if (args.skip.changelog) return;
   await runLifecycleScript(args, 'prechangelog');
-  await outputChangelog(args, newVersion);
+  const changelogOutput = await outputChangelog(args, newVersion);
   await runLifecycleScript(args, 'postchangelog');
+  return changelogOutput;
 }
 
 Changelog.START_OF_LAST_RELEASE_PATTERN = START_OF_LAST_RELEASE_PATTERN;
@@ -95,7 +96,7 @@ function outputChangelog(args, newVersion) {
             '\n' +
             (content + oldContentBody).replace(/\n+$/, '\n'),
         );
-      return resolve();
+      return resolve(content);
     });
   });
 }
